bsnes/target-libretro/libretro.cpp

```diff
@@ -449,7 +470,7 @@ RETRO_API void retro_get_system_info(retro_system_info *info)
 {
 	info->library_name     = "bsnes";
 	info->library_version  = Emulator::Version;
-	info->need_fullpath    = false;
```

Comment:
also, this was out of spec, the core is using the path, and frontends are not required to pass the path when this is false.